lib/image_db.rb

```diff
@@ -26,37 +32,41 @@ def update
     end
 
     # Dribbble
-    data = ImageDB.dribbble_client.get_user('icco').likes
-    data.each {|l| @images.add l.html_url }
+    data = ImageDB.dribbble_client.get_user("icco").likes
+    data.each { |l| @images.add l.html_url }
 
     # Flickr
-    favorites = flickr.favorites.getPublicList(:user_id => '42027916@N00', :extras => 'url_n').map {|p| "http://www.flickr.com/photos/#{p["owner"]}/#{p["id"]}"}
-    favorites.each {|l| @images.add l }
+    favorites = flickr.favorites.getPublicList(user_id: "42027916@N00", extras: "url_n").map { |p| "http://www.flickr.com/photos/#{p['owner']}/#{p['id']}" }
+    favorites.each { |l| @images.add l }
 
     # Write all image links to disk
-    all_images = @images.delete_if {|i| i.empty? }.to_a.sort
-    File.open(Inspiration::LINK_FILE, 'w') {|file| file.write(all_images.to_a.join("\n")) }
+    all_images = @images.delete_if(&:empty?).to_a.sort
+    File.open(Inspiration::LINK_FILE, "w") { |file| file.write(all_images.to_a.join("\n")) }
 
     # VeryGoods.co
-    products = open 'https://verygoods.co/site-api-0.1/users/icco/goods?limit=20' do |j|
+    products = open "https://verygoods.co/site-api-0.1/users/icco/goods?limit=20" do |j|
       data = Oj.compat_load(j)
-      data["_embedded"]["goods"].map {|g| "https://verygoods.co/site-api-0.1#{g["_links"]["product"]["href"]}" }
+      data["_embedded"]["goods"].map { |g| "https://verygoods.co/site-api-0.1#{g['_links']['product']['href']}" }
     end
-    products.each {|p| @images.add p }
+    products.each { |p| @images.add p }
 
-    return true
+    # Instagram
+    ImageDB.instagram_client.user_liked_media.each do |i|
+      @images.add i.link
+    end
+
+    true
   end
 
   def full_update
-
     # Flickr Personal Favorites Set
     # NOTE: Page count verified 2015-07-22
     (1..3).each do |page|
-      p ({ :flickr => '42027916@N00', :set => '72157601200827657', :page => page })
+      p ({ flickr: "42027916@N00", set: "72157601200827657", page: page })
```

Comment:
`(...)` interpreted as grouped expression.<br>Do not write to stdout. Use Rails' logger if you want to log.